go/private/cc.bzl

```diff
@@ -0,0 +1,30 @@
+load("@bazel_tools//tools/cpp:toolchain_utils.bzl", "find_cpp_toolchain")
+
+def new_cc_import(go,
```

Comment:
This should probably be part of `emit_binary` in `go/private/actions/binary.bzl` or at least in that file.